owlbot.py

```diff
@@ -21,5 +21,5 @@
 
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library()
-s.copy(templates)
+s.move(templates, excludes=[".github/auto-label.yaml"])
```

Comment:
is `copy` a wrong way to operate the templates? why do you use `move()`?

Reply:
Both works fine - just copied a line from other source with move